### ISSUE TITLE
How to spawn a building viapython api?

### ISSUE THREAD

**Issue by jeromefan** [carla-simulator/carla#6603]
CARLA version: carla 0.9.14 build from source
Platform/OS: ubuntu 20.04 and win11

Hi, everyone, I want to spawn a building via python api during the simulation.

I know that the buildings in the map is stored as a map layer, (eg. Town 10). But what I'm trying to do is sth like importing a building (.obj file) and use it like `static.prop.mailbox':

```python
mailbox_bp = blueprint_library.find('static.prop.mailbox')
mailbox = world.spawn_actor(mailbox_bp, carla.Transform(carla.Location(z=20)))
```

So, my question seems like to ask how to add a new static.prop.xxx into carla.

**Comment by daohu527** [comment]
Not really, we should use prop factory to add props so that we can call them through python api.
